### ISSUE TITLE
Error because of the 'null' relational fields [graphcool]

### ISSUE THREAD

**Issue by vysakh0** [marmelab/aor-graphql#27]
I've an `User` model that belongs to a `Site` model.  If the User records have proper `siteId`, there is no problem in listing them in `admin on rest`.  When there is a new `User` record with a `null` value in `siteId` field, in the admin on rest app, I get an error `Cannot read property id of null`. 

Is there a way to allow the records with null relations as well? (When the relation is one-one, it is difficult to always the child record value)

**Comment by dervos** [comment]
I've had the same issue.

**Comment by djhi** [comment]
Can you post the error stack ?

**Comment by djhi** [comment]
I suspect the error is [here](https://github.com/marmelab/aor-graphql/blob/master/packages/aor-graphql-client-graphcool/src/getResponseParser.js#L33)

**Comment by djhi** [comment]
I also need a sample data: what does graphcool send you for this user without a site ?

**Comment by dervos** [comment]
Yeah, that could be the error. Graphcool returns a `null` value for a user without a site. `null.id` doesn't exist.

**Comment by djhi** [comment]
Can you guys try with the branch `fix_null_relations` and confirm whether it fixes the prb ?

**Comment by dervos** [comment]
Yes, I can confirm it no longer gives the error and renders fine.

**Comment by djhi** [comment]
![plan](http://s2.quickmeme.com/img/a9/a9ed842f739e930dc8e9340bafbbaeaf77994c50c74fc6a86b046b54cb9b2c59.jpg)

**Comment by djhi** [comment]
Released: https://github.com/marmelab/aor-graphql/releases/tag/v0.0.1-1
